openff/bespokefit/tests/executor/services/qcgenerator/test_worker.py

```diff
@@ -12,8 +12,8 @@ def test_compute_torsion_drive():
     task = Torsion1DTask(
         smiles="[F][CH2:1][CH2:2][F]",
         central_bond=(1, 2),
-        grid_spacing=180,
-        scan_range=(-180, 180),
+        grid_spacing=15,
+        scan_range=(-165, 180),
```

Comment:
The integration test only takes a few minutes even with this change, so this is not a big deal... but could you explain the reasoning behind this change so I can keep it in mind for future tests?